packages/client/components/header/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react'
+import KeyboardArrowLeftIcon from '@material-ui/icons/KeyboardArrowLeft'
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown'
+import styled from 'styled-components'
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+import Popper from '@material-ui/core/Popper'
+import Button from '@material-ui/core/Button'
+import Dialog from '@material-ui/core/Dialog'
+import DialogActions from '@material-ui/core/DialogActions'
+import DialogTitle from '@material-ui/core/DialogTitle'
```

Comment:
This is fine. But on side note, you can import all of them from `@material-ui/core`. (`import { Popper, Button, .... } from @material-ui/core`)